demisto_sdk/commands/common/errors.py

```diff
@@ -515,6 +516,11 @@ def no_docker_tag(docker_image):
         return f'{docker_image} - The docker image in your integration/script does not have a tag.' \
                f' Please create or update to an updated versioned image\n'
 
+    @staticmethod
+    @error_code_decorator
+    def non_existing_docker(docker_image):
+        return f'{docker_image} - Could not find the docker image. Are you sure it exists on DockerHub?'
```

Comment:
```suggestion
        return f'{docker_image} - Could not find the docker image. Check if it exists in DockerHub.'
```

If you can provide a link to where should the user check that would be great.